# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 366245552cae
# Full Hash: 366245552cae536ef883be806322395dc47b759d
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-06-17 09:24:05
# Regressor Bug: 1951038
# File Overlap Count: 1
# Description:
#   Bug 1951038 - Get rid of the legacy white-space normalizer of `HTMLEditor` and the pref r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D253589
# ==============================================================================

diff -r b13b0a530402 -r 366245552cae modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Jun 17 00:10:27 2025 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Jun 17 00:37:22 2025 +0000
@@ -5780,13 +5780,6 @@
   value: false
   mirror: always
 
-# Whether enabling blink compatible white-space normalizer or keep using
-# Gecko's traditional white-space normalizer.
-- name: editor.white_space_normalization.blink_compatible
-  type: bool
-  value: true
-  mirror: always
-
 # General prefs for editor, indicating whether Gecko-specific editing UI is
 # enabled by default. Those UIs are not implemented by any other browsers.  So,
 # only Firefox users can change some styles with them. This means that Firefox